app/reducers/index.js

```diff
@@ -15,7 +15,11 @@ const defaultState = {
     up: 0,
     down: 0
   },
-  inspect: { key: null }
+  inspect: { key: null },
+  titleUnderEdit: {
```

Comment:
this name isn't clear to me, `under` is mostly used for location.

Can we call this for example `titleEditInPlace`?

Reply:
@juliangruber yeah `titleEditInPlace` sounds better. Making that change now....

Reply:
My - horrible - default approach would be `currentlyEditedTitle`. To me both `titleEditInPlace` and `titleUnderEdit` are almost equal so I leave it up to you. Other ideas: `focussedTitle` or `titleEditorField`

Reply:
`currentlyEditedTitle` works for me too, it's natural +1 but different from all the other state names -1

Reply:
Since `titleEditInPlace` is already in this PR. lets go with that 😆